drivers/place/auto_release.cr

```diff
@@ -221,6 +224,24 @@ class Place::AutoRelease < PlaceOS::Driver
     self[:released_booking_ids] = [] of Int64
   end
 
+  def template_fields : Array(TemplateFields)
+    # TODO: add more fields to be consistent with other templates
```

Comment:
More useful fields needs to be added to this driver as it's missing most of the fields available to other drivers.